relay/blockchain/exchange_events.py

```diff
@@ -0,0 +1,55 @@
+from eth_utils import force_bytes, is_hex, decode_hex
+from .events import TLNetworkEvent, BlockchainEvent  # NOQA
+
+LogFillEventType = 'LogFill'
+LogCancelEventType = 'LogCancel'
+
+
+class ExchangeEvent(TLNetworkEvent):
+
+    def __init__(self, web3_event, current_blocknumber, timestamp, user=None):
+        super().__init__(web3_event, current_blocknumber, timestamp, from_to_types, user)
+        self.exchange_address = web3_event.get('address')
+        if (is_hex(web3_event.get('args').get('orderHash'))):
```

Comment:
Short comment why this can be hex. Event can be from our database where it is saved as a hex string. If there is an issue, also put a ref to it.